dojson/utils.py

```diff
@@ -30,7 +30,7 @@ def int_with_default(value, default):
 
 
 def ignore_value(f):
-    """Remove key for None value.
+    """Remove key for None or empty value value.
```

Comment:
It is my understanding that TIND added some real-life like test records, including the empty string field value use case.  @audub  @Kennethhole @remileduc @PXke can you confirm this is still wanted?

@kaplun Note that altering existing test cases should be done only *very* carefully, ideally after an RFC, because the tests were presumably entered for a good reason, and altering their behaviour based on one Invenio service's wishes can break other Invenio services' use cases.  This is one of the things that integrator check during merge:  http://invenio.readthedocs.io/en/latest/processes/integrating.html#beware-of-inter-service-relations